packages/autorest.go/src/m4togocodemodel/clients.ts

```diff
@@ -34,27 +34,27 @@ export function adaptClients(m4CodeModel: m4.CodeModel, codeModel: go.CodeModel)
 
```

Comment:
Is everything being switched to use go.MethodType? See line 32 still uses the old way

Reply:
joel fixed in 861d4fc018912c79c8db2693be8c5093666d7f86